libethereum/ClientTest.cpp

```diff
@@ -60,8 +60,9 @@ void ClientTest::setChainParams(string const& _genesis)
         if (params.sealEngineName != "NoProof")
             BOOST_THROW_EXCEPTION(ChainParamsNotNoProof() << errinfo_comment("Provided configuration is not well formatted."));
 
+        setAuthor(params.author);  // Must be called before reopenChain !!!
         reopenChain(params, WithExisting::Kill);
-        setAuthor(params.author); //for some reason author is not being set
+        completeSync();  // set sync state to idle for mining
```

Comment:
These seems to be a bugfix?

Reply:
yes. it resolve setAuthor issue

Reply:
Do you want to create a new PR for the bugfix?

Reply:
yes

Reply:
Ping 😉

Reply:
https://github.com/ethereum/cpp-ethereum/pull/4922

Reply:
Thanks!